source/connect/csot.txt

```diff
@@ -1,210 +1,241 @@
 .. _pymongo-csot:
 
-Client Side Operation Timeout
-=============================
+Limit Server Execution Time
+===========================
 
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 1
+   :depth: 2
    :class: singlecol
 
 .. facet::
    :name: genre
    :values: reference
  
 .. meta::
-   :keywords: error, blocking
+   :keywords: error, blocking, thread, task
 
-PyMongo 4.2 introduced the ``~pymongo.timeout()`` method and the ``timeoutMS``
-URI and keyword argument to the ``~pymongo.mongo_client.MongoClient`` class.
-These features allow applications to more easily limit the amount of time that
-one or more operations can execute before control is returned to the app. This
-timeout applies to all of the work done to execute the operation, including
-server selection, connection checkout, serialization, and
-server-side execution.
+Overview
+--------
 
-Basic Usage
------------
+When you use {+driver-short+} to perform a server operation, you can also limit the amount
```

Comment:
```suggestion
When you use {+driver-short+} to perform a server operation, you can limit the amount
```

Reply:
shoot, good catch. probably shouldn't have been working on this while watching basketball... 🏀